packages/core/src/messages/responses/cache-sorted-set-increment-score.ts

```diff
@@ -20,7 +20,14 @@ import {ResponseBase, ResponseError, ResponseSuccess} from './response-base';
  * }
  * ```
  */
-export abstract class Response extends ResponseBase {}
+export abstract class Response extends ResponseBase {
+  public score(): number | undefined {
+    if (this instanceof Success) {
```

Comment:
Is the implementation missing from this one? it looks like you might just have the abstract class implemented.

Reply:
good eye - the implementation already existed though so it doesn't show up in this diff.